### PR TITLE
Issue 220: storage.hbase.short-cf-names is not fixed

### PR DESCRIPTION
**Pull request by jerryjch**
Check and warn if short-cf-names is interleaved or mixed with the same HBase table.

### CODE REVIEW

**Comment by jerryjch** [comment]
I was thinking to throw an exception. But it then would be an 'incompatible' change.
Just issuing a warning for now.

File: janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java [hunk 1]
```diff
@@ -759,6 +759,23 @@ private HTableDescriptor ensureTableExists(String tableName, String initialCFNam
...
             if (adm.tableExists(tableName)) {
                 desc = adm.getTableDescriptor(tableName);
+                // Check and warn if long and short cf names are mixedly used for the same table.
+                if (shortCfNames && initialCFName.equals(SHORT_CF_NAME_MAP.get(SYSTEM_PROPERTIES_STORE_NAME))) {
+                    String longCFName = SHORT_CF_NAME_MAP.inverse().get(initialCFName);
+                    if (desc.getFamily(longCFName.getBytes()) != null) {
+                        logger.warn("Configuration {}=true, but the table \"{}\" already has column family with long name \"{}\".",
+                            SHORT_CF_NAMES.getName(), tableName, longCFName);
+                        logger.warn("Check {} configuration.", SHORT_CF_NAMES.getName());
+                    };
```
Comment (amcp) [inline janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java]: minor remove semicolon here

File: janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java [hunk 2]
```diff
@@ -759,6 +759,23 @@ private HTableDescriptor ensureTableExists(String tableName, String initialCFNam
...
+                }
+                else if (!shortCfNames && initialCFName.equals(SYSTEM_PROPERTIES_STORE_NAME)) {
+                    String shortCFName = SHORT_CF_NAME_MAP.get(initialCFName);
+                    if (desc.getFamily(shortCFName.getBytes()) != null) {
+                        logger.warn("Configuration {}=false, but the table \"{}\" already has column family with short name \"{}\".",
+                            SHORT_CF_NAMES.getName(), tableName, shortCFName);
+                        logger.warn("Check {} configuration.", SHORT_CF_NAMES.getName());
+                    };
```
Comment (amcp) [inline janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java]: minor remove semicolon here

File: janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java (hunk 1 above)
Comment (jerryjch) [inline janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java]: Will fix.

File: janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java (hunk 2 above)
Comment (jerryjch) [inline janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java]: Will fix.

**Comment by jerryjch** [comment]
Let me see what I can do on the test.

**Comment by jerryjch** [comment]
@amcp Mind taking a look at the new PR?

**Comment by amcp** [comment]
Looks good!

**Pull request merged**